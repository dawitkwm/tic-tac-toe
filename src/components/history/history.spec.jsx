import React from "react";
import { shallow } from "enzyme";

import History from "./history";

describe("History component", () => {
  it("should render History component", () => {
    const historyWrapper = shallow(<History />);
    expect(historyWrapper).toMatchSnapshot();
  });
});
