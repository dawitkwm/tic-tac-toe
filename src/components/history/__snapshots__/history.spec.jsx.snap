// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History component should render History component 1`] = `
<div>
  <p>
    History
  </p>
  <div>
    <button
      key="1"
    >
      Move #
      1
    </button>
  </div>
</div>
`;
